scripts/st2bootstrap-deb.sh

```diff
@@ -617,7 +617,7 @@ install_st2mistral() {
   sudo crudini --set /etc/mistral/mistral.conf database connection "postgresql+psycopg2://mistral:${ST2_POSTGRESQL_PASSWORD}@127.0.0.1/mistral"
 
   # Setup Mistral DB tables, etc.
-  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head | grep -v "ironicclient" || true
```

Comment:
I'm :+1: for filtering those warnings out, but :-1: for `|| true` construct which will suppress real mistral DB upgrade failures.

I think following similar approach we did before would work OK.
See: https://github.com/StackStorm/st2-packages/pull/486/files